lib/support/permissions/ability.rb

```diff
@@ -18,6 +18,7 @@ def initialize(user)
 
         can :read, :anonymous_feedback
         can :request, :global_export_request if user.has_permission?('feedex_exporters')
+        can :request, :review_feedback if user.has_permission?('feedex_exporters')
```

Comment:
Yeah, this role is pretty bogusly named now. We should ask someone to add a new role to signon for the support app and use that instead. Perhaps `feedback_reviewers`.